trigger/project_triggers.go

```diff
@@ -65,6 +65,24 @@ func TriggerDownstreamVersion(args ProcessorArgs) (*model.Version, error) {
 	if err != nil {
 		return nil, err
 	}
+	settings, err := evergreen.GetConfig()
+	if err != nil {
+		return nil, errors.Wrap(err, "error getting evergreen settings")
+	}
+	upstreamProject, err := model.FindOneProjectRef(args.SourceVersion.Identifier)
```

Comment:
Should still check that upstreamProject != nil

Reply:
fixed